megamek/src/megamek/client/ui/swing/ClientDialog.java

```diff
@@ -51,6 +53,12 @@ public ClientDialog(JFrame owner, String title, boolean modal) {
         super(owner, title, modal);
         this.owner = owner;
     }
+    
+    public ClientDialog(JFrame owner, String title, boolean modal, boolean scale) {
```

Comment:
Is it possible to get some explanation on why somebody would use one of these constructors over the others?